lambda/custom/state_handlers.js

```diff
@@ -74,10 +74,21 @@ var state_handlers = {
       },
       FinishedArticle: function() {
         console.log('START_MODE:FinishedArticle');
+        if (this.attributes['full']) {
+          scout_agent
+            .updateArticleStatus(
+              this.attributes['userId'],
+              this.attributes['articleId'],
+              0
+            )
+            .catch(function(err) {
+              console.log('Error during offset update.');
```

Comment:
In catch blocks like this, dump the contents of `err` too so we can get some idea of what went wrong.